msteams-platform/webhooks-and-connectors/how-to/add-outgoing-webhook.md

```diff
@@ -98,6 +101,29 @@ Responses from your outgoing webhooks appear in the same reply chain as the orig
     "text": "This is a reply!"
 }
 ```
+### Adaptive Card example response
+
+```csharp
+private static Attachment GetAdaptiveCardAttachment()
```

Comment:
I think this could be more helpful/accessible if this example was JSON instead of C# code, for me at least. Anyone who isn't using the C# SDK will not be able to learn from this.